web/app/components/header/account-setting/model-provider-page/model-parameter-modal/parameter-item.tsx

```diff
@@ -127,18 +127,7 @@ const ParameterItem: FC<ParameterItemProps> = ({
       && !isNullOrUndefined(parameterRule.min)
       && !isNullOrUndefined(parameterRule.max)
 
-    if (parameterRule.type === 'int' || parameterRule.type === 'float') {
-      let step = 100
-      if (parameterRule.max) {
-        if (parameterRule.max < 10)
-          step = 0.1
-        else if (parameterRule.max < 100)
-          step = 1
-        else if (parameterRule.max < 1000)
-          step = 10
-        else if (parameterRule.max < 10000)
-          step = 100
-      }
+    const step = parameterRule.type === 'int' ? 1 : 0.1;
```

Comment:
These lines should not be removed, I think there should be a better way to do this.